foundations/likvid-prod-alz/platforms/azure/base/terragrunt.hcl

```diff
@@ -0,0 +1,26 @@
+include "platform" {
+    path = find_in_parent_folders("platform.hcl")
+    expose = true
+  }
+
+include "module" {
+    path = find_in_parent_folders("module.hcl")
+  }
+
+terraform {
+    source = "${get_repo_root()}//kit/alz/base"
+  }
+
+inputs = {
+    
+  root_parent_id = "likvid-prod"
```

Comment:
d: Is there a reason why we have alz as an intermediate MG? Why not have likvid as root_parent and likvid-prod as root_id/root_name?

Reply:
Sure, Thats a better Idea. I will change it in the next PR